pynubank/nubank.py

```diff
@@ -203,6 +203,16 @@ def get_account_balance(self):
         data = self._make_graphql_request('account_balance')
         return data['data']['viewer']['savingsAccount']['currentSavingsBalance']['netAmount']
 
+    @requires_auth_mode(AuthMode.APP)
+    def get_account_savings_main_balance(self):
```

Comment:
Poderia ser o mesmo método que já retorna as duas funções né ? Assim quem for recuperar as informações de saldo faz uma chamada só para a API